lib/workos/user_management.rb

```diff
@@ -214,24 +214,37 @@ def create_user(email:, password: nil, first_name: nil, last_name: nil, email_ve
       # Update a user
       #
       # @param [String] id of the user.
-      # @param [String] first_name The user's first name.
-      # @param [String] last_name The user's last name.
-      # @param [Boolean] email_verified Whether the user's email address was previously verified.
+      # @param [Hash] update_hash
+      # @option update_hash [String] first_name The user's first name.
+      # @option update_hash [String] last_name The user's last name.
+      # @option update_hash [Boolean] email_verified Whether the user's email address was previously verified.
+      # @option update_hash [String] password The user's password.
+      # @option update_hash [String] password_hash The user's hashed password.
+      # @option update_hash [String] password_hash_type The algorithm originally used to hash the password.
+      #  Valid values are bcrypt.
       sig do
         params(
           id: String,
-          first_name: T.nilable(String),
-          last_name: T.nilable(String),
-          email_verified: T.nilable(T::Boolean),
+          update_hash: {
+            first_name: T.nilable(String),
+            last_name: T.nilable(String),
+            email_verified: T.nilable(T::Boolean),
+            password: T.nilable(String),
+            password_hash: T.nilable(String),
+            password_hash_type: T.nilable(String),
+          },
         ).returns(WorkOS::User)
       end
-      def update_user(id:, first_name: nil, last_name: nil, email_verified: nil)
+      def update_user(id:, update_hash:)
```

Comment:
Is there a reason we are switching to collecting all of the previous keyword arguments into a new `update_hash` parameter? 

This would be a breaking change. It also seems less idiomatic in Ruby than adding new keyword arguments for the password-related fields.

Reply:
It's one of our Rubocop rule. We can't have 6 parameters in a function.

Feels ok to ship this breaking change since the current one has not yet been released in a stable version.

Reply:
Ah, gotcha. We have other places where we have 6 or more keyword arguments, like here for SSO:

https://github.com/workos/workos-ruby/blob/f88b01a9ded91723d947038b079e965ff6a73a88/lib/workos/sso.rb#L71-L81

In those cases we explicitly disable the rule with `rubocop:disable Metrics/ParameterLists`, but I think we should instead update our `.rubocop.yml` to exclude keyword arguments entirely from this rule, and then remove those magic comments:

```yml
# .rubocop.yml

Metrics/ParameterLists:
  Max: 6
  CountKeywordArgs: false
```

Then we can go back to adding more keyword arguments to the `update_user` method. Does that seem reasonable?

Reply:
reasonable! I update the PR.

I like this rule tho...